units/systemd-tpm2-clear.service.in

```diff
@@ -0,0 +1,25 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Issue TPM Clear Request
+Documentation=man:systemd-tpm2-clear.service(8)
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=tpm2.target
```

Comment:
This does not have dependencies to /dev/tpm0 (only have /dev/tpmrm0).
But the service writes an sysattr of /dev/tpm0, rather than /dev/tpmrm0.
I think it is better to add dependencies to /dev/tpm0 in this service or tpm2.target.

Reply:
true. will add to tpm2.target